.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,36 +1,28 @@
-<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
-
 ## About the PR
-<!-- What did you change? -->
-
-## Why / Balance
-<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
-
-## Technical details
-<!-- Summary of code changes for easier review. -->
+<!-- What did you change? Why? -->
 
 ## Media
 <!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
-Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
+Small fixes/refactors are exempt. -->
 
 ## Requirements
 <!-- Confirm the following by placing an X in the brackets [X]: -->
-- [ ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
+- [ ] I have tested all added content and changes.
 - [ ] I have added media to this PR or it does not require an ingame showcase.
-<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->
+
+## Technical details
+<!-- Summary of code changes for easier review. -->
 
 ## Breaking changes
 <!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
 This will be posted in #codebase-changes. -->
 
 **Changelog**
 <!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
-Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
-Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
-<!--
+Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog.
+Add your name after the :cl: to have it credited in game. -->
```

Comment:
```suggestion
Add your name after the :cl: to change the display name credited in game, otherwise your github name will be used. Delete the changelog if you are leaving it empty. -->
```